lib/client.js

```diff
@@ -16,10 +16,12 @@ class Client {
    * @param {String} config.baseUrl ISS for FHIR server
    * @param {Object} config.customHeaders Optional custom headers to send with
    *   each request
+   * @param {String} config.key TLS key for FHIR server
+   * @param {String} config.cert TLS certificate for FHIR server
```

Comment:
These are optional so we should have `[]`.

Reply:
Fixed :)

Reply:
Or I guess say “Optional” like the param.